modules/setting/lfs.go

```diff
@@ -36,7 +36,9 @@ func loadLFSFrom(rootCfg ConfigProvider) {
 	// DEPRECATED should not be removed because users maybe upgrade from lower version to the latest version
 	// if these are removed, the warning will not be shown
 	deprecatedSetting(rootCfg, "server", "LFS_CONTENT_PATH", "lfs", "PATH", "v1.19.0")
-	lfsSec.Key("PATH").MustString(sec.Key("LFS_CONTENT_PATH").String())
+	if sec.HasKey("LFS_CONTENT_PATH") {
```

Comment:
This will not create `LFS_CONTENT_PATH`.

Reply:
Well, it creates .... that's the ini package's quirk ....

![image](https://github.com/go-gitea/gitea/assets/2114189/0dc80016-4b18-400f-8fd3-d33fe7949366)

Reply:
> Well, it creates .... that's the ini package's quirk ....
> 
> ![image](https://user-images.githubusercontent.com/2114189/244404822-0dc80016-4b18-400f-8fd3-d33fe7949366.png)

Oh, God! I previously think only `MustXXX` will create something.